hook.py

```diff
@@ -27,3 +27,10 @@
                                'Disclaimer: this may cause duplicate links to run.',
                    module='plugins.stockpile.app.obfuscators.base64_jumble')
     )
+    await stockpile_svc.data_svc.store(
+        Obfuscator(name='caesar cipher',
+                   description='A shifting caesar cipher algorithm to obfuscate commands before running them. The '
+                               'shift value can only be accessed before a command has been executed, making it '
```

Comment:
this description is off a bit-- this is what i originally intended when i wrote the prototype but this PR is (slightly) different. in the prototype, i assigned a random shift character to the link (i.e., the pin). then, i added a "one-time use" REST API endpoint that the agent would hit asking for the link's pin. any subsequent hits on the API endpoint would = None (to prevent a defender from retrieving the shift key). 

in this PR, the shift key is in the open, inside the command. this is perfectly fine-- but we should update the description here.

Reply:
Done, I have updated the description.